client/src/ui/atoms/icon/index.scss

```diff
@@ -8,7 +8,7 @@ $icons: "add-filled", "add", "altname", "bell-filled", "bell", "bookmark-filled"
   "prefix", "preview", "previous", "quote", "safari", "samsunginternet",
   "search", "send", "server", "sidebar", "simple-firefox", "small-arrow",
   "theme-light", "star-filled", "star", "theme-os-default", "thumbs-down",
-  "thumbs-up", "trash", "twitter", "unknown", "warning", "webview", "yes",
+  "thumbs-up", "trash", "twitter", "mastodon", "unknown", "warning", "webview", "yes",
```

Comment:
This list is (mostly) alphabetically ordered, can you add "mastodon" between "language" and "menu-filled", please?

Reply:
Thanks for your review 🙂

Moved the place of `"mastodon"` string.